x/lockup/keeper/store.go

```diff
@@ -5,8 +5,9 @@ import (
 	"fmt"
 	"time"
 
+	errorsmod "cosmossdk.io/errors"
+
```

Comment:
Thank you for the review. Fixed it here: [9cf3817](https://github.com/dymensionxyz/dymension/pull/1278/commits/9cf3817a509882312a3a411727fe718b4ae137a9)

Just to mention, there are several files in the codebase that doesn't follow this 3 section rule. Please consider opening an issue regarding that. so someone can work on that and we can this rule uniformly applied throughout the codebase.

Maybe it's the good idea to also mention this in the `Contributing.md` file. It will help new contributors to follow the coding guidelines for this repository while contributing.